front/src/components/boxs/clock/Clock.jsx

```diff
@@ -0,0 +1,152 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import dayjs from 'dayjs';
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import { default as ReactClock } from 'react-clock';
+dayjs.extend(localizedFormat);
+
+import { CLOCK_TYPES } from '../../../../../server/utils/constants';
+import { Text } from 'preact-i18n';
+
+const padding = {
+  paddingLeft: '20px',
+  paddingRight: '20px',
+  paddingTop: '10px',
+  paddingBottom: '10px'
+};
+
+const Clock = ({ children, ...props }) => (
+  <div class="card">
+    {props.clockType === CLOCK_TYPES.ANALOG && (
+      <div
+        style={{
+          display: 'flex',
```

Comment:
There is too many style, why don't you store element styling in css file?

Reply:
Yes you have right, I'm fixing that :)